requirements.txt

```diff
@@ -0,0 +1,3 @@
+wget
```

Comment:
Is not better to replace by python3-wget ?

Reply:
Probably a better idea, `wget` is a bit older, but it works on my system. python3_wget works fine now on my end, too.